docs/web-apps/automated-testing/cdp.md

```diff
@@ -176,6 +179,20 @@ For more information, please refer to the [Playwright Selenium Grid documentatio
 
 - CDP / BiDi Sessions are currently limited to 10mins
 - Extended debugging cannot be used along CDP / BiDi
+- CDP /BiDi is NOT available on legacy OS versions
```

Comment:
Maybe it's worth mentioning that Selenium 4 is required (not our limitation, but could be useful for the customer)